### ISSUE TITLE
Support private Hosted packages please

### ISSUE THREAD

**Issue by mk-wang** [luanpotter/vscode-dart-import#13]
I'm using a private  hosed package.
Pubspec.yaml is something like https://dart.dev/tools/pub/dependencies#hosted-packages mentioned:

```
dependencies:
  transmogrify:
    hosted:
      name: transmogrify
      url: http://your-package-server.com
    version: ^1.4.0
```

After running 'Fix Imports' Cmd, I got
> Expected to find a single line starting with 'name:' on pubspec.yaml file, 2 found.

**Comment by luanpotter** [comment]
One thing has nothing to do with the other; the packages is looking for your own package name, not your dependencies. You are missing the name line for your app/package.

**Comment by luanpotter** [comment]
Fixed by #17
